index.html

```diff
@@ -228,6 +228,22 @@ <h2 class="about-header">About Myself</h2>
           </li>
         </ul>
       </section>
+      <section class="contact">
+        <div class="contact-wrap d-flex">
+          <div class="contact-intro d-flex">
+            <h2 class="contact-header">Contact me</h2>
+            <p class="contact-text">if you have an application you are interested in developing, a feature that you need built or
+              a project that needs coding. I'd love to help with it!</p>
+          </div>
+          <form class="contact-form d-flex">
```

Comment:
- Hey, let's ensure our form uses the action attribute and the post method to integrate with [formspree service](https://formspree.io/html) to make the form submittable. kindly fix this.